Enterprise-Knowledge-Graph-Corporate/db_scripts/queries/company_holders.gsql

```diff
@@ -1,6 +1,7 @@
-CREATE QUERY CompanyHolders(vertex<Company> company, uint step) FOR GRAPH EnterpriseGraph syntax v2 {
+CREATE QUERY company_holders(vertex<Company> company, uint step) 
+  FOR GRAPH EnterpriseGraph syntax v1 {
   /**
```

Comment:
/** -> /*

Reply:
I'm going to leave this bc it seems intentional and is a common way of commenting code